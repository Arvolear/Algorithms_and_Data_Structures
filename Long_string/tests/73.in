97640060811053674935319389883
62349355558753033536289347328884119