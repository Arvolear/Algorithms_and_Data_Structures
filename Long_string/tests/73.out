62349453198813844589964282648274002
62349257918692222482614412009494236
6087794868286653675440835959616894477798281318607375926287968077
638563